[2022-03-23 20:16:41,972] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: dbt_dag.test.iowa.staging_sales manual__2022-03-23T19:14:12.667416+00:00 [queued]>
[2022-03-23 20:16:41,979] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: dbt_dag.test.iowa.staging_sales manual__2022-03-23T19:14:12.667416+00:00 [queued]>
[2022-03-23 20:16:41,979] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-23 20:16:41,979] {taskinstance.py:1244} INFO - Starting attempt 2 of 2
[2022-03-23 20:16:41,979] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-23 20:16:41,986] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): test.iowa.staging_sales> on 2022-03-23 19:14:12.667416+00:00
[2022-03-23 20:16:41,988] {standard_task_runner.py:52} INFO - Started process 356 to run task
[2022-03-23 20:16:41,991] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dbt_dag', 'test.iowa.staging_sales', 'manual__2022-03-23T19:14:12.667416+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/transform_dags.py', '--cfg-path', '/tmp/tmpz7jlrqp3', '--error-file', '/tmp/tmp20lo0pud']
[2022-03-23 20:16:41,991] {standard_task_runner.py:77} INFO - Job 109: Subtask test.iowa.staging_sales
[2022-03-23 20:16:42,036] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_dag.test.iowa.staging_sales manual__2022-03-23T19:14:12.667416+00:00 [running]> on host a91ac75a8f43
[2022-03-23 20:16:42,080] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dbt_dag
AIRFLOW_CTX_TASK_ID=test.iowa.staging_sales
AIRFLOW_CTX_EXECUTION_DATE=2022-03-23T19:14:12.667416+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-03-23T19:14:12.667416+00:00
[2022-03-23 20:16:42,080] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-23 20:16:42,084] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            cd /opt/iowa/dbt/iowa &&\n            dbt --no-write-json test --target dev --models staging_sales\n            ']
[2022-03-23 20:16:42,088] {subprocess.py:85} INFO - Output:
[2022-03-23 20:16:44,710] {subprocess.py:89} INFO - 20:16:44  Running with dbt=1.0.4
[2022-03-23 20:16:44,748] {subprocess.py:89} INFO - 20:16:44  Found 1 model, 0 tests, 0 snapshots, 0 analyses, 188 macros, 0 operations, 0 seed files, 1 source, 0 exposures, 0 metrics
[2022-03-23 20:16:44,749] {subprocess.py:89} INFO - 20:16:44
[2022-03-23 20:16:44,749] {subprocess.py:89} INFO - 20:16:44  [WARNING]: Nothing to do. Try checking your model configs and model specification args
[2022-03-23 20:16:46,089] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-23 20:16:46,111] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=dbt_dag, task_id=test.iowa.staging_sales, execution_date=20220323T191412, start_date=20220323T201641, end_date=20220323T201646
[2022-03-23 20:16:46,142] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-23 20:16:46,168] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
